pkgs/development/libraries/libmkv/default.nix

```diff
@@ -15,6 +15,11 @@ stdenv.mkDerivation rec {
   preConfigure = "sh bootstrap.sh";
 
   meta = {
+    description = "A stalled alternative lightweight Matroska muxer written for HandBrake";
```

Comment:
What does "stalled" mean in this context? No longer maintained (last commit 7 years old)?

Reply:
Yes.

Reply:
What a better word to use, or use "no longer maintained"?

Reply:
Fossilized seems too hard in this context, since libraries are still in use by several projects.
I am not a native English speaker, so maybe there is a better suggestion.

Reply:
I think "unmaintained" gets the point across. Arguably, unmaintained media libraries should have a stronger warning (such as being marked insecure), given that they're likely to parse potentially malicious data, but that's strictly irrelevant to this PR.

Reply:
I am trying to determine term once and for all.

Since we maintain unmaintained library, this term seems misleading.
"Unmaintained library, provides functions to read, create, and modify mp4 files"
Strange use of words. After all it was maintained in the main project, and it is maintained here.

"No longer maintained library..." seems more appropriate.
"Library no longer maintained upstream" seems to the right direction.
"No longer developed" seems to point to the more directly.
Well, "fossilized" may be right after all
"Abandoned" seems like the most versatile scary term.
Also probably word "project" would direct to the central point.
"Abandonware" is the right term, but its meaning is grabbed to relate to proprietary software. If abandonware is GPL, that also means no one can press charges.

I would probably settle for "Adandoned", but I myself would start to use term "Abandonware" for all licenses, and propose to anyone to use this term.

Reply:
Yep: [Wikipedia: Abandonware](https://en.wikipedia.org/wiki/Abandonware) one of meanings: "Open source and freeware programs that have been abandoned"